Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -2227,6 +2475,64 @@ def main() -> None:
         elif command == "expanse-list-risk-rules":
             return_results(list_risk_rules_command(client, demisto.args()))
 
+        elif command == "expanse-get-services":
+            return_results(get_services_command(client, demisto.args()))
+
+        elif command == "expanse-get-service":
+            return_results(get_service_command(client, demisto.args()))
+
+        elif command == "expanse-list-pocs":
+            return_results(list_pocs_command(client, demisto.args()))
+
+        elif command == "expanse-create-poc":
+            return_results(create_poc_command(client, demisto.args()))
+
+        elif command == "expanse-assign-pocs-to-asset":
+            args = demisto.args()
+            args['operation_type'] = 'ASSIGN'
+            return_results(manage_asset_pocs_command(client, demisto.args()))
```

Comment:
```suggestion
            return_results(manage_asset_pocs_command(client, args)
```

I think you meant to send here and in the following commands - args instead of demisto.args()